tap_dynamics/client.py

```diff
@@ -207,7 +212,10 @@ def _make_request(self, method, endpoint, paging=False, headers=None, params=Non
         return results
 
     def get(self, endpoint, paging=False, headers=None, params=None):
-        return self._make_request("GET", endpoint, paging, headers=headers, params=params)
+        try:
+            return self._make_request("GET", endpoint, paging, headers=headers, params=params)
+        except DynamicsException as e:
+            raise SymonException(f'Import failed with the following MS Dynamics error: {str(e)}','dynamics.DynamicsApiError')
```

Comment:
example:
`tap_dynamics.symon_exception.SymonException: Import failed with the following MS Dynamics error: {"error":{"code":"0x80060888","message":"EntityMetadata With Id = LogicalName='account1' does not exist."}}`